resolwe/test/testcases.py

```diff
@@ -281,7 +281,7 @@ def setUp(self):
         self._register_schemas()
 
         self.collection = Collection.objects.create(contributor=self.admin, name="Test collection")
-        self.files_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'files')
+        self.files_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'toolkit', 'tests', 'files'))
```

Comment:
Before it was even worse. The `self.files_path` pointed to a folder that did not exist at all. I suggest we keep my change until we have process tests in two places.